examples/lists-and-named-requests/src/action-creators/get-latest-books.js

```diff
@@ -5,32 +5,32 @@ export default function getLatestBooks() {
   return function(dispatch) {
     dispatch({
       type: actionTypes.READ_RESOURCES_PENDING,
-      resourceName: 'books',
-      request: 'readLatestBooks',
-      list: 'latestBooks'
+      resourceType: 'books',
+      requestKey: 'readLatestBooks',
+      list: 'latestBooks',
     });
```

Comment:
Note: the old action creators here do still work. They were just using deprecated action properties :v: